Fibonacci_Number/Fibonacci_NumberDP.java

```diff
@@ -11,22 +11,22 @@ int fibDP(int n)
 	    return n;	
 	}
 	    
-        /* This is used to print the values or series of given fibonacci number */
-        /*if(dp[n] != 0)
+        /* This is used to print the values or series of given fibonacci number.
+        if(dp[n] != 0)
 	{
             return dp[n];
 	}*/
 	
 	/* This recursion is to hold the values till cand1 = 1(from 5-4-3-2-1), cand2 = 1(5-3-1) */
 	int cand1 = fibDP(n-1);
 	int cand2 = fibDP(n-2);
-	/* Later, it will go back to its previous values(2-3), cand1, cand2 values will be updated. */
-	/* after, getting values (1,1), those values get added and updated at 5th index. this will repeat in each iteration */
-	/* you can check(print) manually the values of cand1, cand2, n and dp[n] at multiple instances to get a proper idea of structure of the program */
+	/* Later, it will go back to its previous values(2-3), cand1, cand2 values will be updated.
+	   after, getting values (1,1), those values get added and updated at 5th index. this will repeat in each iteration,
+	   you can check(print) manually the values of cand1, cand2, n and dp[n] at multiple instances to get a proper idea of structure of the program.
```

Comment:
break the into another line

Reply:
breaked into another line... Please check now.